homeassistant/components/netatmo/switch.py

```diff
@@ -80,3 +106,70 @@ async def async_turn_off(self, **kwargs: Any) -> None:
         await self.device.async_off()
         self._attr_is_on = False
         self.async_write_ha_state()
+
+
+PERSONS_ENTITY_DESCRIPTION_KEY = "presence"
+
+
+class NetatmoPersonHomeSwitch(NetatmoDeviceEntity, SwitchEntity):
+    """Representation of a Netatmo person home/away switch."""
+
+    person: Person
+    _attr_configuration_url = CONF_URL_SECURITY
+
+    def __init__(self, person: NetatmoPerson) -> None:
+        """Initialize the Netatmo device."""
+        super().__init__(person.data_handler, person.person)
+        self.person = person.person
+        self._signal_name = f"{HOME}-{self.home.entity_id}"
+        self._publishers.extend(
+            [
+                {
+                    "name": HOME,
+                    "home_id": self.home.entity_id,
+                    SIGNAL_NAME: self._signal_name,
+                },
+            ]
+        )
+        self.entity_description = SwitchEntityDescription(
+            key=PERSONS_ENTITY_DESCRIPTION_KEY, device_class=SwitchDeviceClass.SWITCH
+        )
+        self._attr_device_info = DeviceInfo(
+            identifiers={
+                (DOMAIN, f"{person.parent_id}-{PERSONS_DEVICE_IDENTIFIER_SUFFIX}")
+            },
+            name=f"{self.home.name} {PERSONS_DEVICE_NAME_SUFFIX}",
+            manufacturer=self.device_description[0],
+            model=self.device_description[1],
+            configuration_url=self._attr_configuration_url,
+        )
+        self._attr_name = self.person.pseudo
+        self._attr_unique_id = (
+            f"{self.person.entity_id}-{PERSONS_ENTITY_DESCRIPTION_KEY}"
```

Comment:
```suggestion
            f"{person.entity_id}-{PERSONS_ENTITY_DESCRIPTION_KEY}"
```
How unique are the entity_ids of the person?

Reply:
Netatmo persons IDs are UUIDs